### ISSUE TITLE
Markdown not being rendered properly.

### ISSUE THREAD

**Issue by axyut** [b0sc/b0sc.github.io#30]
- [X] PR for this issue must follow the PR template
- [x] This is a bug
- [x] This is a feature request

## For Bug 
- What happened?
  - 
![image](https://github.com/user-attachments/assets/83c4380d-c3f7-4982-bfa5-643540095a06)

- What should have happened?
  - markdown renders `#` for `h1` tag but due to some bug, be it from astro or tailwind, the h1 tag is as same as h6, not only in the blogs page but everywhere on the site, you can test in `jobs.astro` file.
  - also change the default font for the markdown rendering or the blogs page
  - also change ordered and unordered lists and use of tilde ``
  - basically make markdown rendering as same as in the github, like this comment currently.

**Comment by dinesh-58** [comment]
Headings are [unstyled by default](https://tailwindcss.com/docs/preflight#headings-are-unstyled) in tailwind and have `font-size: inherit`. Can be fixed by using [`@layer base` directives](https://tailwindcss.com/docs/adding-custom-styles#adding-base-styles)

**Comment by BIIJESH** [comment]
Thanks @dinesh-58
